docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -327,255 +388,102 @@ In Geo-Partitioning, the table must be splitted into partitions, and each partit
+  PARTITION p2 VALUES LESS THAN (2010),
+  PARTITION p3 VALUES LESS THAN (2015),
+  PARTITION p4 VALUES LESS THAN MAXVLUE PLACEMENT POLICY='storeonfastssd'
+ )
+PLACEMENT POLICY='companystandardpolicy';
 ```
 
-This statement defines placement rules for one partition, including its local indices.
-
-The key format of a partitioned table is `t_{partition_id}_r_{pk_value}`. As `partition_id` is part of the key prefix, the key range of a partition is successive. The key range is `t_{partition_id}_` to `t_{partition_id+1}_`.
-
-Placement rules can also be defined on a partitioned table. Because there are multiple key ranges for the table, multiple rules will be generated and sent to PD. When placement rules are defined both on the table and its partitions, the rule priorities described later should be applied.
-
-### Unpartitioned index placement
-
-Defining placement rules of indices is more complicated, because indices can be unpartitioned or partitioned. Each case should be considered separately.
-
-The index here can be primary index or secondary index. When the key of a clustered index is `_tidb_rowid` rather than the primary key, the primary index is actually an unclustered index. In this case, an index placement statement is applied.
-
-Expression indices and invisible indices are also supported, as the key format is the same as normal.
-
-Defining placement of an unpartitioned index is in such a statement:
+In this example, partition `p0` uses the policy `storeonhdd`, partition `p4` uses the policy `storeonfastssd` and the remaining partitions use the policy `companystandardpolicy`. Assuming the following `ALTER TABLE` statement is executed, only partitions `p1`-`p3` will be updated:
 
 ```sql
-ALTER TABLE table_name ALTER INDEX index_name
-	{ADD | ALTER} PLACEMENT POLICY CONSTRAINTS=constraints ROLE=role REPLICAS=replicas,
-	...
-	
-ALTER TABLE table_name ALTER INDEX index_name
-	DROP PLACEMENT POLICY ROLE=role,
-	...
+ALTER TABLE t1 PLACEMENT POLICY=`companynewpolicy`;
 ```
 
-This key format of an unpartitioned index is `t_{table_id}_i_{index_id}_r_{pk_value}`. The key range can be inferred by `table_id` and `index_id`.
-
-### Partitioned index placement
+This behavior is inspired by how a `CHARACTER SET` or `COLLATE` attribute applies to a column of a table, and columns will use the character set defined at the table-level by default.
 
-Defining placement rules of an index in one specific partition is in such a statement:
+#### Removing placement from a database, table or partition
 
+Placement policy can be removed from an object via the following syntax:
 ```sql
-ALTER TABLE table_name ALTER PARTITION partition_name ALTER INDEX index_name
-	{ADD | ALTER} PLACEMENT POLICY CONSTRAINTS=constraints ROLE=role REPLICAS=replicas,
-	...
-	
-ALTER TABLE table_name ALTER PARTITION partition_name ALTER INDEX index_name
-	DROP PLACEMENT POLICY ROLE=role,
-	...
+ALTER DATABASE test [DEFAULT] PLACEMENT=default;
```

Comment:
```suggestion
ALTER DATABASE test [DEFAULT] PLACEMENT SET DEFAULT;
```

This would be more in line with `ALTER TABLE ... ALTER COLUMN ... SET DEFAULT` and might also allow future things like `SET INACTIVE` etc.